.github/workflows/publish.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish Package
+
+on:
+  release:
+    types: [published]
+
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install node
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
+        registry-url: 'https://registry.npmjs.org'
+    - name: Install Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip build twine
+    - name: Publish the Python package
+      env:
+        TWINE_USERNAME: __token__
+        TWINE_PASSWORD: ${{ secrets.PYPI_API_TOKEN }}
+      run: |
+        python -m build
```

Comment:
I did not know about that package - thanks

Quick question, it seems that sdist and wheel are not activated by default (cf [documentation](https://pypa-build.readthedocs.io/en/stable/)). Should it be activated?

Reply:
I think that it implicitly works thanks to the build_meta specified in the pyproject.toml (see https://setuptools.readthedocs.io/en/latest/build_meta.html#how-to-use-it); the documentation is a bit scarce on this indeed (see https://github.com/pypa/setuptools/issues/1698)

Reply:
Thanks for documentation pointers.